src/renate/updaters/experimental/repeated_distill.py

```diff
@@ -229,7 +229,7 @@ def __init__(
         super().__init__(
             model=model,
             memory_size=memory_size,
-            memory_batch_size=batch_size,
+            batch_memory_frac=-1.0,
```

Comment:
is it not using the memory?

Reply:
It is *only* sampling from the memory using the entire batch size. So passing a dummy value to the parent class (which probably means this inheritance structure is not ideal...)